### ISSUE TITLE
FilePicker.PickMultipleAsync Does not work on Xiaomi phones, still can only select one, not multiple

### ISSUE THREAD

**Issue by cnchenkg** [dotnet/maui#18998]
This my code 
```       
FileTypes = new FilePickerFileType(new Dictionary<DevicePlatform, IEnumerable<string>>() {  
            { DevicePlatform.Android, new[] { "text/plain" } },
        { DevicePlatform.WinUI, new[] { ".txt" } },}),
    };
    try
    {
       var result = await FilePicker.PickMultipleAsync(options);
      
        if (result != null)
        {
           AddItems(result);
        }

    }
```

When I start selecting, I should be able to select multiple options, but the file browser only allows me to select one

**Comment by drasticactions** [comment]
Not having a Xiaomi device, I can't test this directly, but have you tried holding down on an item to select it? I believe that's the behavior to enable multiple selections within file pickers, where you then can select multiple items.

**Comment by cnchenkg** [comment]
> Not having a Xiaomi device, I can't test this directly, but have you tried holding down on an item to select it? I believe that's the behavior to enable multiple selections within file pickers, where you then can select multiple items.


Yes, I tried, it didn't work on Xiaomi devices, in addition I succeeded on the android emulator of VS2022, it can be seen that it should just not work properly on Xiaomi devices, because it is a MIUI system and not a native Android system, and when it is called, it is the MIUI file manager of the device system to select.
![屏幕截图 2023-11-24 212250](https://github.com/dotnet/maui/assets/124700390/6730749c-ee04-4f1c-b2f1-2228923405c9)
